src/libs/actions/Search.ts

```diff
@@ -32,7 +42,19 @@ function search({hash, query, policyIDs, offset, sortBy, sortOrder}: SearchParam
 
     API.read(READ_COMMANDS.SEARCH, {hash, query, offset, policyIDs, sortBy, sortOrder}, {optimisticData, finallyData});
 }
-export {
-    // eslint-disable-next-line import/prefer-default-export
-    search,
-};
+
+function createTransactionThread(hash: number, transactionID: string, reportID: string, moneyRequestReportActionID: string) {
```

Comment:
Might be good to add a doc here to explain what we use this for.

Reply:
Added!